SwiftyContextMenu/ContextMenuBackgroundBlurView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ContextMenuBackgroundBlurView.swift
+//  SwiftyContextMenu
+//
+//  Created by Paul Bancarel on 28/11/2020.
+//
+
+final class ContextMenuBackgroundBlurView : IntensityVisualEffectView {
+    
+    private let style: ContextMenuUserInterfaceStyle
+    
+    init(_ style: ContextMenuUserInterfaceStyle) {
+        self.style = style
+        super.init(effect: ContextMenuBackgroundBlurView.blurEffect(style), intensity: 0.25)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        effect = ContextMenuBackgroundBlurView.blurEffect(style)
+    }
+    
+    fileprivate static func blurEffect(_ style: ContextMenuUserInterfaceStyle) -> UIBlurEffect {
+        switch style {
+        case .automatic:
+            if #available(iOS 13.0, *) {
+                return UIBlurEffect(style: .light)
+            } else {
+                fatalError("Cannot have an automatic blur effect below iOS 13.")
```

Comment:
wait, instead of making the app crash, can't we print and return some default value?

Reply:
I made the change but normally it's impossible to pass throught that code because `.automatic` is iOS13 + due to:
```
public enum ContextMenuUserInterfaceStyle {
    @available(iOS 13, *) case automatic
    case light, dark
}
```
But at compile time swift doesn't know that. If i'm in a `switch case .automatic` then I'm necessarily on iOS 13+. Swift asks me to check but it's useless.
Do you have an idea to change the implementation in order to make the compiler understand that ?